### ISSUE TITLE
[BUG] -I cannot install the package

### ISSUE THREAD

**Issue by Khalilsqu** [ikun-svelte/ikun-ui#347]
### Describe the bug

I keep getting this error when trying to install the package.

### System Info

```Shell
found 0 vulnerabilities
npm ERR! code ERESOLVE
npm ERR! ERESOLVE could not resolve
npm ERR!
npm ERR! While resolving: @ikun-ui/core@0.0.13-beta.3
npm ERR! Found: unocss@0.57.7
npm ERR! node_modules/unocss
npm ERR!   dev unocss@"^0.57.7" from the root project
npm ERR!
npm ERR! Could not resolve dependency:
npm ERR! peer unocss@"^0.56.5" from @ikun-ui/core@0.0.13-beta.3
npm ERR! node_modules/@ikun-ui/core
npm ERR!   @ikun-ui/core@"^0.0.13-beta.3" from the root project
npm ERR!
npm ERR! Conflicting peer dependency: unocss@0.56.5
npm ERR! node_modules/unocss
npm ERR!   peer unocss@"^0.56.5" from @ikun-ui/core@0.0.13-beta.3
npm ERR!   node_modules/@ikun-ui/core
npm ERR!     @ikun-ui/core@"^0.0.13-beta.3" from the root project
npm ERR!
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
npm ERR!
npm ERR!
npm ERR! For a full report see:
npm ERR! C:\Users\kalho\AppData\Local\npm-cache\_logs\2023-11-26T13_43_11_510Z-eresolve-report.txt

npm ERR! A complete log of this run can be found in: C:\Users\kalho\AppData\Local\npm-cache\_logs\2023-11-26T13_43_11_510Z-debug-0.log
```

### Used Package Manager

npm

### Validations

- [X] Read the [Contributing Guidelines](https://github.com/ikun-svelte/ikun-ui/blob/main/CONTRIBUTING.md).
- [X] Read the [docs](https://ikun-ui.netlify.app/).
- [X] Check that there isn't [already an issue](https://github.com/ikun-svelte/ikun-ui/issues) that reports the same bug to avoid creating a duplicate.
- [X] Make sure this is a ikun-ui issue and not a framework-specific issue. For example, if it's a Svelte SFC related bug, it should likely be reported to https://github.com/sveltejs/svelte instead.
- [X] The provided reproduction is a [minimal reproducible example](https://stackoverflow.com/help/minimal-reproducible-example) of the bug.

**Comment by baiwusanyu-c** [comment]
This is caused by the unocss version in peerDependencies being 0.56.5. The temporary solution is to specify the unocss version used in the project. This problem will be fixed later.

```
{
  "name": "test",
  "version": "1.0.0",
  "description": "",
  "main": "index.js",
  "scripts": {
    "test": "echo \"Error: no test specified\" && exit 1"
  },
  "author": "",
  "license": "ISC",
  "dependencies": {
    "@ikun-ui/core": "^0.0.13-beta.3"
  },
  "devDependencies": {
    "@ikun-ui/preset": "^0.0.13-beta.3",
    "unocss": "^0.56.5"
  }
}

```

**Comment by baiwusanyu-c** [comment]
fixed: 0e1e3fca1bce171f055e546b63b8b8aa48889ae7
